lib/telemetry-application-rs/src/lib.rs

```diff
@@ -434,11 +444,7 @@ fn telemetry_resource(config: &TelemetryConfig) -> Resource {
     // TODO(fnichol): create opentelemetry-resource-detector-aws for ec2 & eks detection
     Resource::from_detectors(
         Duration::from_secs(3),
-        vec![
```

Comment:
@fnichol I wasn't sure what these were for exactly. The `ProcessResource` and `OsResource` detectors no longer seem to be available.

Reply:
If I recall, the `ProcessResouce` gave us attributes like `process.command_args` and `process.pid`, and the `OsResource` gave attributes such as `os.type` (maybe that was about it?). There also used to be one for AWS which I never enabled (at the time I wasn't sure of our deployment strategy). Maybe these were split out or retired--maybe I can check as well.